tests/test-data/ialirt_ec2_primary/Dockerfile

```
@@ -13,8 +21,24 @@ WORKDIR /app
 # Install Flask
 RUN pip install flask
 
+# Non-interactive frontend for apt-get
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install dependencies and mountpoint-s3
+RUN apt-get update && \
+    apt-get install -y sudo wget fuse apt-utils && \
+    wget https://s3.amazonaws.com/mountpoint-s3-release/latest/x86_64/mount-s3.deb && \
+    sudo apt-get install -y ./mount-s3.deb && \
+    rm ./mount-s3.deb
+
+# Create the /mnt/s3 directory
+RUN mkdir -p /mnt/s3
```

Remove an image layer here.
```suggestion
    rm ./mount-s3.deb && \
    # Create the /mnt/s3 directory
    mkdir -p /mnt/s3
```